src/data/services/LmsApiService.js

```diff
@@ -87,6 +88,12 @@ class LmsApiService {
       return Promise.reject(result);
     });
   }
+
+  static fetchGradeOverrideHistory(subsectionId, userId) {
+    let historyUrl = `${LmsApiService.baseUrl}/api/grades/v1/subsection/${subsectionId}/?user_id=${userId}`;
+    historyUrl += '&history_record_limit=5';
```

Comment:
Please add these strings together in the initial statement and use `const` rather than `let`